.travis.yml

```diff
@@ -26,6 +27,9 @@ install:
   - chmod +x ./travis.sh
   - make
   - make emake
+  - curl -L https://github.com/fundies/enigma-libs/files/1347135/enigma-libs.zip > enigma-libs.zip
```

Comment:
We should probably rehost these somewhere more stable, at some point.

Reply:
Ya. I havent finished getting everything to build. Alure is being a pain. I was going to give you a zip to host  once I build alure and all of gtk's 200 dependancies and etc